BritneyS/SparkleSpin
# Add style assets

## What you did :question:
For this PR, I added the style assets for the project: Fonts, App Icons, Colors, and Favicons (Home, Delete, and Exit).

## How to test it :microscope:
Run the app to see the added Fonts, as well as asset folders 'Assets', 'Colors' and 'Icons'.

## Any background context you want to provide? :question:
Checking Apple's guidelines, an 'Exit' or 'Quit' button might not be necessary. If there's no need for it in the design, I will remove the 'Exit' favicon.

The 'Delete' favicon may also be changed.

## Screenshots (if applicable) :camera:
![screen shot 2019-02-20 at 2 57 50 pm](https://user-images.githubusercontent.com/8409475/53122356-6e788e00-3524-11e9-8920-50476b854b28.png)
![screen shot 2019-02-20 at 2 58 12 pm](https://user-images.githubusercontent.com/8409475/53122367-733d4200-3524-11e9-8cda-30d530a8e8a4.png)
![screen shot 2019-02-20 at 2 58 51 pm](https://user-images.githubusercontent.com/8409475/53122370-77695f80-3524-11e9-8e39-af86380c1006.png)
![screen shot 2019-02-20 at 2 58 58 pm](https://user-images.githubusercontent.com/8409475/53122381-7e906d80-3524-11e9-808c-33c87c0bf345.png)
![screen shot 2019-02-20 at 3 34 38 pm](https://user-images.githubusercontent.com/8409475/53122647-14c49380-3525-11e9-9d26-c8ee9311e480.png)
